src/pages/ProhibitorySigns/components/ScenarioWizard/ScenarioWizard.test.tsx

```diff
@@ -6,6 +6,8 @@ import { getPathTo } from '../../../../routes'
 import { withApp } from '../../../../../test/utils/withApp'
 
 describe('ScenarioWizard', () => {
+  jest.setTimeout(10000)
```

Comment:
Why do we need to wait 10 seconds? (comment ook voor andere plekken waar dit gebeurd)

Reply:
Many tests often failed within the default 5 second timeout. By doubling that amount for affected tests no more timeouts occur (at least on my machine).

This is not really a robust solution, but there are many issues with these tests and they should be properly looked at, at some point.

Reply:
So, I'm not going to fix the tests properly in this PR, but I'll add an issue to the backlog.